qt/aqt/data/web/css/webview.scss

```diff
@@ -1,26 +0,0 @@
-/* Copyright: Ankitects Pty Ltd and contributors
- * License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html */
-
-@use 'ts/sass/core';
-@use 'ts/sass/scrollbar';
-@use 'ts/sass/buttons';
-
-// core.scss sets border-box, but we need to
-// keep the old behaviour for now to avoid breaking
-// add-ons/card templates
-* {
-  box-sizing: content-box;
-}
```

Comment:
The screens that show card templates like the reviewer and clayout seem to be using border-box now, which I suspect will break some card templates

Reply:
(aside from that, looks good :-)

Reply:
Hm, honestly, I might have not quite thought over the consequences of changing the base CSS for reviewer.css as well.
Including Reboot into the reviewer css will have more effects than just box-sizing, like removing top margins from `<p>`, giving lists a left padding, etc..

I think most users upon this PR would just notice that "everything looks a little bit better now", but users with intricate card templates which are not bootstrap-compatible will certainly complain.

I could revert the change for reviewer.css, and set up a new PR, making this choice opt-in through the Preferences, and we give users a few versions to adapt their templates.

What do you think?

Reply:
Ugh, good point. Unfortunately users may be using card templates from a shared deck published years ago and no longer maintained by the deck author, so I fear we'll need to support the old styling for a long time. Can we start by unblocking this PR, either by changing reviewer.scss to not include reset, or by rolling back the changes to that file?

Reply:
Re a flag, the review area is largely the domain of the user rather than Anki's UI, so I wonder whether changing the default styling there is really worth it? But assuming it is, maybe we could do so with a flag on the template - that way new templates could default to the new styling, with old templates using the older styling. It would probably mean we'd need to inline the reset css into the .card div, and not sure if that would have any effect on performance or not.